pkg/network/ebpf/c/protocols/events-types.h

```diff
@@ -26,6 +26,7 @@ typedef struct {
 } batch_key_t;
 
 typedef struct {
+    __u32 cpu;
```

Comment:
Wondering if we truly need 32b to represent a CPU. I think u16 should suffice, and it will reduce (a little bit) the struct size, and the maps related to it.

Also, I didn't see in the PR description the "additional cost" of the `*_batches` maps (http_batches, etc.) size due to the change of the struct, can you please add it?